src/main/java/org/duraspace/bagit/serialize/TarGzBagSerializer.java

```diff
@@ -37,15 +35,13 @@ public Path serialize(final Path root) throws IOException {
             final TarArchiveOutputStream tar = new TarArchiveOutputStream(gzip)) {
             tar.setLongFileMode(TarArchiveOutputStream.LONGFILE_POSIX);
             tar.setBigNumberMode(TarArchiveOutputStream.BIGNUMBER_POSIX);
-            final List<Path> files = Files.walk(root).collect(Collectors.toList());
+            final List<Path> files = Files.walk(root).toList();
```

Comment:
Same issue as `TarBagSerializer`, just need to move the Stream init to the try w/ resources.